src/item.h

```diff
@@ -974,6 +976,8 @@ class item : public visitable<item>
          */
         std::string durability_indicator( bool include_intact = false ) const;
 
+        int dirt = 0;
```

Comment:
Nearly no item will need this. It should be an item variable (see `item::set_var` and `item::get_var`).

Reply:
I searched for that but not sure I see any examples I could copy.

Reply:
I keep getting dirt is not a member of item despite having done it.set_var("dirt",0);